File: SaveTheDate.DL/Models/Guest.cs
using System;
using System.Collections.Generic;

#nullable disable

namespace SaveTheDate.DL.Models
{
    public partial class Guest
    {
        public Guest()
        {
            EventGifts = new HashSet<EventGift>();
        }

        public int Id { get; set; }
        public int UserId { get; set; }
        public int EventId { get; set; }
        public bool ArrivalConf { get; set; }
        public int? TableNum { get; set; }
        public int? BusId { get; set; }
        public int? GiftId { get; set; }

        public virtual Bus Bus { get; set; }
        public virtual Event Event { get; set; }
        public virtual EventGift Gift { get; set; }
        public virtual Table TableNumNavigation { get; set; }
        public virtual User User { get; set; }
        public virtual ICollection<EventGift> EventGifts { get; set; }
    }
}

File: SaveTheDateAngular/src/app/Classes/Guest.ts
import { Bus } from "./Bus";
import { User } from "./User";
import { Event } from "./Event";
import { EventGift } from "./EventGift";
import { Table } from "./Table";


export class Guest{
 
         Id?: number;
         UserId!: number|undefined;
         EventId!:number|undefined;
         ArrivalConf!:boolean;
         TableNum?: number;
         BusId?: number;
         GiftId?: number;

         Bus?:Bus;
         Event?:Event;
         EventGift?:EventGift
         Table?:Table
         User?:User;
      
         constructor(){}
    };


// export class Guest{
//     constructor(
//         public Id: number,
//         public UserId: number,
//         public EventId:number,
//         public ArrivalConf:boolean,

//         public TableNum?: number,
//         public  BusId?: number,
//         public  GiftId?: number
//     ){}
//     };




    
       